WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -2,84 +2,84 @@
+ bookmark_476_odoacer: "Odovacar"
+ bookmark_476_odoacer_desc: "Odovacar is a citizen and general of Rome, and member of a Germanic #italic foederati#!. When the #foederati#! were denied lands in Italy, Odovacar led them against Romulus Augustus, displacing the boy emperor and founding the Kingdom of Italy. Thoroughly an integrated Roman citizen, his efforts were met with praise and reward from Zeno, the Emperor-in-the-East, as he was allowed to govern Italy as an independent Roman province. However, left without the support of the Eastern Roman Empire, Odovacar must meet the threats that for so long had destabilized the Western Roman Empire.\n\n#bold Can Odovacar secure a place for his people, or will he meet the same end that befell other Roman pretenders?#!"
  
  #Empire in Crisis
  bm_476_empire_in_crisis: "Empire in Crisis"
  bm_476_empire_in_crisis_desc: "Forced to flee Constantinople on January 9, 475, former Emperor Zeno now challenges the Usurper Basiliskos for control of Constantinople with the support of Illus and his rebel troops. In New Rome, the Miaphysite Emperor Basiliskos has steadily eroded his base of support and sits precariously on the throne. Will the former Emperor successfully retake the Crown, or will the Usurper successfully hold the crown?\n\nThe Caucasus remain a hotbed of contention between Rome and #italic Ērān#!, threatening to erupt at any moment.\n\nEven beyond the borders of the Empire, the erstwhile Huns could consolidate and pose yet another threat to Rome.\n\n#bold Both Old and New Rome are in Crisis, and it will take a great emperor to save Rome from others and itself...#!"
  
- bookmark_476_emperor_zeno:0 "Zeno"
- bookmark_476_emperor_zeno_desc:0 "Zeno Tarasis would reign as the Emperor of the Eastern Roman empire during a period filled of religious turmoil and unrest. After losing the title of emperor, he initiates a war to regain his lost empire against the Miaphysite Emperor Basiliskos. Although, even with victory here, harsher threats must still be faced. Hordes of Germanic barbarians roam the empire, causing unrest and chaos, while Persia merely waits for their next chance to finish Byzantium.\n\n#bold Will the East fall, just the same as the West?#!"
+ bookmark_476_emperor_zeno: "Zeno"
+ bookmark_476_emperor_zeno_desc: "Zeno, born an Isaurian named Tarasis, would reign as the Emperor of the Eastern Roman empire during a period filled of religious turmoil and unrest. After losing the title of emperor to a palace coup, he initiated a war to regain his lost empire against the Miaphysite Emperor Basiliskos. Although, even with victory here, more threats must still be faced. Hordes of Germanic barbarians roam the empire, causing unrest and chaos, while Persia merely waits for their next chance to finish Rome.\n\n#bold Will the East fall, just the same as the West?#!"
```

Comment:
I agree with the change but I would use ERE as opposed to Rome. Or Constantinople in that case but I'm not sure

Reply:
I'd go with Constantinople, because there are still two Romes at this start date, so you're right, saying Rome gives a very false impression.